components/support/restmail-client/src/lib.rs

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use error::RestmailClientError;
```

Comment:
nit: We could do a `type Result<T, E = RestMailClientError> = Result<T, E>;` Then throughout the file just use `Result<T>`